src/e2e_tests/common/pages.py

```diff
@@ -201,6 +215,20 @@ def active_executor_tab(self):
     def add_new_tab_button(self):
         return self.find_element(".tab [title='Add another script instance']")
 
+    @property
+    def history_table(self):
+        return self.find_element("div.executions-log-table tr")
```

Comment:
Why do you return "tr" as a "history table" ? tr is only a single row
Please either remove the tr from here or rename the method name